helpers/result.go

```diff
@@ -0,0 +1,24 @@
+package helpers
+
+import (
+	"go-google-scraper-challenge/models"
+)
+
+func PrepareResultSet(results []*models.Result) [][]*models.Result {
```

Comment:
Is this the function to make the view display 2 tables in single page? If yes, I think this one should be `presenter` instead of helpers because it is the function to display some style in the view. 

You can check Junan or Hoang repo for more information about presenter

What do you think?

Reply:
✅  updated on [57ca4d5](https://github.com/carryall/go-google-scraper-challenge/pull/68/commits/57ca4d516aa2ad2b6fcb0da2c2e19f5b1418d592)